Pipfile

```diff
@@ -0,0 +1,24 @@
+[[source]]
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+Django = "*"
+python-dotenv = "*"
+django-heroku = "*"
+google-api-python-client = "*"
+social-auth-app-django = "*"
+django-taggit = "*"
+url-normalize = "*"
+
+[dev-packages]
+pre-commit = "*"
+black = "*"
+coverage = "*"
```

Comment:
Is this Pipfile auto-generated? Can these packages be alphabetized?

Reply:
Yeah, it's autogenerated. For whatever reason, running pipenv install <x> does not alphabetize the Pipfile.
I will look into a solution for this.

Reply:
I'm planning to update the dockerfile and ci before landing this

Reply:
+1 on updating the Docker config :)